These changes, including the config.h theme, were taken from:
https://github.com/karahobny/acme2k
(acme2k makes other changes to Acme which aren't in this patch.)

acme2k licensed their "modest modifications" to Acme (which would include the
contents of this patch) under an MIT license, which can be found here:
https://github.com/karahobny/acme2k/blob/master/docs/LICENSE.MIT

To use with plan9port acme:
% cd $PLAN9
% patch -p1 < acme-dark-theme.patch
% cd src/cmd/acme
% mk install

diff --git a/src/cmd/acme/acme.c b/src/cmd/acme/acme.c
index d001a2a8..cd5c3f44 100644
--- a/src/cmd/acme/acme.c
+++ b/src/cmd/acme/acme.c
@@ -11,6 +11,7 @@
 #include <libsec.h>
 #include "dat.h"
 #include "fns.h"
+#include "config.h"
 	/* for generating syms in mkfile only: */
 	#include <bio.h>
 	#include "edit.h"
@@ -1039,19 +1040,17 @@ iconinit(void)
 	Image *tmp;
 
 	if(tagcols[BACK] == nil) {
-		/* Blue */
-		tagcols[BACK] = allocimagemix(display, DPalebluegreen, DWhite);
-		tagcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DPalegreygreen);
-		tagcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DPurpleblue);
-		tagcols[TEXT] = display->black;
-		tagcols[HTEXT] = display->black;
-
-		/* Yellow */
-		textcols[BACK] = allocimagemix(display, DPaleyellow, DWhite);
-		textcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DDarkyellow);
-		textcols[BORD] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DYellowgreen);
-		textcols[TEXT] = display->black;
-		textcols[HTEXT] = display->black;
+		tagcols[BACK]	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TAGBG);
+		tagcols[HIGH]	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TAGHLBG);
+		tagcols[BORD]	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_COLBUTTON);
+		tagcols[TEXT]	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TAGFG);
+		tagcols[HTEXT]	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TAGHLFG);
+
+		textcols[BACK] 	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TXTBG);
+		textcols[HIGH] 	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TXTHLBG);
+		textcols[BORD] 	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_SCROLLBG);
+		textcols[TEXT] 	= allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TXTFG);
+		textcols[HTEXT] = allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TXTHLFG);
 	}
 
 	r = Rect(0, 0, Scrollwid+ButtonBorder, font->height+1);
@@ -1075,15 +1074,15 @@ iconinit(void)
 	r.max.x -= ButtonBorder;
 	border(modbutton, r, ButtonBorder, tagcols[BORD], ZP);
 	r = insetrect(r, ButtonBorder);
-	tmp = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DMedblue);
+	tmp = allocimage(display, Rect(0,0,1,1), RGBA32, 1, C_TMPBUTTON);
 	draw(modbutton, r, tmp, nil, ZP);
 	freeimage(tmp);
 
 	r = button->r;
-	colbutton = allocimage(display, r, screen->chan, 0, DPurpleblue);
+	colbutton = allocimage(display, r, RGBA32, 1, C_WINBUTTON);
 
-	but2col = allocimage(display, r, screen->chan, 1, 0xAA0000FF);
-	but3col = allocimage(display, r, screen->chan, 1, 0x006600FF);
+	but2col = allocimage(display, r, screen->chan, 1, C_BUTTON2HL);
+	but3col = allocimage(display, r, screen->chan, 1, C_BUTTON3HL);
 }
 
 /*
diff --git a/src/cmd/acme/config.h b/src/cmd/acme/config.h
new file mode 100644
index 00000000..1d5c6b10
--- /dev/null
+++ b/src/cmd/acme/config.h
@@ -0,0 +1,36 @@
+/*
+ *  colors-constants. now let me take a minute to explain
+ *  them. C_TAGBG/FG/HLBG/HLFG are respectively your tag
+ *  windows background, foreground, highlighted back- and
+ *  foreground color. The same exact applies to C_TXT*-
+ *  variants of the same constant just in this case it handles
+ *  the colorscheme of the text window.
+ *
+ *  C_WINBUTTON is the button which you adjust your columns
+ *  position with. C_COLBUTTON represents a separate text
+ *  windows button. C_TMPBUTTON is the so-called "dirty"
+ *  or "unclean" -marker aka notifying the file as modified put
+ *  in the middle of C_COLBUTTON. C_SCROLLBG is plain and simple
+ *  your scrollbars background color.
+ *
+ *  the last two defines are for defining the color of the specific
+ *  buttons highlight background.
+ */
+
+#define C_TAGBG 0x2B303BFF
+#define C_TAGFG 0xC0C5CEFF
+#define C_TAGHLBG 0x006CA5FF
+#define C_TAGHLFG 0xEFF1F5FF
+
+#define C_TXTBG 0x2B303BFF
+#define C_TXTFG 0xC0C5CEFF
+#define C_TXTHLBG 0x006CA5FF
+#define C_TXTHLFG 0xEFF1F5FF
+
+#define C_WINBUTTON 0x543F5EFF
+#define C_COLBUTTON 0x65737EFF
+#define C_TMPBUTTON 0x65737EFF
+#define C_SCROLLBG 0x222222FF
+
+#define C_BUTTON2HL 0xB47EADFF
+#define C_BUTTON3HL 0xBF616AFF
diff --git a/src/cmd/acme/mkfile b/src/cmd/acme/mkfile
index 18bea9e0..2bad712c 100644
--- a/src/cmd/acme/mkfile
+++ b/src/cmd/acme/mkfile
@@ -29,6 +29,7 @@ OFILES=\
 HFILES=dat.h\
 	edit.h\
 	fns.h\
+	config.h
 
 <$PLAN9/src/mkone
 <$PLAN9/src/mkdirs
